packages/elements/src/react/utils/generate-password-error-text.ts

```diff
@@ -18,6 +18,7 @@ const errorMessages: Record<keyof Omit<ComplexityErrors, 'allowed_special_charac
 };
 
 export type ErrorMessagesKey = Autocomplete<keyof typeof errorMessages>;
+export type ErrorCodeOrTuple = ErrorMessagesKey | [ErrorMessagesKey, Record<string, string | number>];
```

Comment:
If we move forward with the "validations" nomenclature, this could be `PasswordValidation`.